source/sync/config-table.rst

```diff
@@ -89,6 +89,11 @@
      - A list of field names to use as :ref:`global queryable fields
        <queryable-fields>`.
 
+   * - | ``queryable_fields_version``
```

Comment:
should the `config.json` example be updated to include this field? ...also, what even is the version of a queryable field?

Reply:
According to @jonatlee (thanks!), we should remove `queryable_fields_version` as it's not editable by the user. Removing it

Reply:
Yeah for some more context, it is an internal field that we use to track when people make changes to their queryable fields. we shouldn't expose it to end users (we intentionally ignore any import requests coming from the cli with it set)